client/quantum_serverless/core/decorators.py

```diff
@@ -317,3 +321,116 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+ENTRYPOINT_CONTENT = """
+import cloudpickle
+from quantum_serverless import get_arguments, save_result
+
+arguments = get_arguments()
+
+with open("./pickle.pkl", "rb") as file:
+    function = cloudpickle.load(file)
+
+    result = function(**arguments)
+    if result is not None:
+        save_result(result)
+"""
+
+
+def distribute_program(
+    provider: Optional[Any] = None,
+    dependencies: Optional[List[str]] = None,
+    working_dir: Optional[str] = None,
+):
+    """[Experimental] Program decorator to turn function into remotely executable program.
+
+    Example:
+        >>> @program(provider=Provider(...), dependencies=[...])
```

Comment:
@distributed_program?

Reply:
well, good question: it is `@distributed_program` or `@distribute_program` here? 😂

Reply:
we have `distribute_task`, therefore I'm using `distribute_program` :)

Reply:
well, that close the question quite easily 😂

Reply:
Then, the `Example` must be match to the name :-)